README.md

```diff
@@ -1,5 +1,81 @@
 # IWC - Intergalactic Workflow Commission
 
+## This group exists to create a registry of community workflows. 
+
+### Reproducibility is important. Our goals are to:
+ - foster workflow use
+ - incorporate versioning
+ - capture more metadata: (names, versions, authors, use cases, etc.)
+ - help scientists find workflows!
+
+### Methods Proposal
+ - discover workflows
+ - establish quality standards (version numbers, tools exist, author is set, training available, test exists ...)
+ - automated testing of workflows utilizing [Planemo](https://github.com/galaxyproject/planemo) test
+ - two layers of test data:
+   - basic functionality testing (quick to run, tools exist) 
+   - downsampled data (validity of workflow results)
```

Comment:
I agree that we should have some example analysis with the full details, but we're not there yet. The rest is again redundant with the contributing document